rtf_vis_tool/src/Components/CoordinateGrid.js

```diff
@@ -20,41 +25,41 @@ const getGridLines = () => {
     return finalLineGridSet;
 }
 
-//Component which renders the x,y,z axes along with the grid lines along xy plane
+// Component which renders the x,y,z axes along with the grid lines along the xy plane.
+// Used within Data_Association_PC.js
+//  This component returns the format JSX, which is essentially a modified version of HTML  
+//  that is meant to be readable by React code. 
 const CoordinateGrid = () => { 
     return (
       <mesh>
           <Line 
-            points={[[-10,0,0], [10,0,0]]} 
+            points={[[lowerGridBound,0,0], [upperGridBound,0,0]]} 
             color="red" 
-            position={[0,0,0]}
             lineWidth={0.3}
           />
-          <Html scaleFactor={50} position={[10,0,0]}>
+          <Html scaleFactor={htmlSize} position={[upperGridBound,0,0]}>
             <div class="content">
               x
             </div>
           </Html>
 
           <Line 
-            points={[[0,-10,0], [0,10,0]]} 
+            points={[[0,lowerGridBound,0], [0,upperGridBound,0]]} 
```

Comment:
it looks like we are naming "y" as "z". Could you add a comment with a link to the React documentation explaining why we do this?

Reply:
yup, I'll add some more info